test/xpu/extended/skip_list_common.py

```diff
@@ -11,6 +11,9 @@
     # 6. std functions get different results when input is nan or inf between GCC and SYCL.
     "test_compare_cpu_cumsum_xpu_bfloat16",
     "test_compare_cpu_cumsum_xpu_float16",
+    "test_compare_cpu_logcumsumexp_xpu_bfloat16",
```

Comment:
May I know failed details for these cases ?

Reply:
Found
- accuracy issue on bfloat16 `Greatest absolute difference: 0.03125 at index (1, 4, 2) (up to 0.001 allowed)`
- nan issue on complext64. But complex128 is OK with the same input.
```
logcumsumexp_complex_xpu_complex64 

input : [1e3 + 0j, 1e-18 + 1e4j, 1e2 + 1e-8j] 
cpu_output :  [1000.+0.j, 1000.+0.j, 1000.+0.j]
cuda_output : [1000.+0.j, 1000.+0.j, 1000.+0.j]
xpu_output : [1000.+0.j, 1000.+0.j, nan + nanj]
```

It seems that xpu scan kernel causes some accuracy issue in logcumsumexp. It may be related to accumulated order.  Still debugging.

Reply:
We may need to open an issue to track them ? @xytintel May I have your comments here ?

Reply:
You can create an issue and link to this PR, and leave the issue link in the code. Notice: 1. CUDA wasn't tested for such bf16 comparison case,  2. CPU bf16 result is not golden reference..

Reply:
Thanks @xytintel and @majing921201.

After doing some experiment, I found that the **`nan` issue in complex64 is caused by accumulated order**: our xpu scan kernel would firstly reduce `input[1], input[2]`, then reduce `input[0], input[2]` in this case.  However, even cpu kernel will output `nan, nanj` when directly calculating `logcumsumexp(input[1], input[2])`. Offline synced with YuanFeng, we can skip this case.

As for bf16 and complex128, the accuracy difference is tolerable and may be related to accumulated order, too. I will skip them and create an issue.